app/src/main/res/layout/partial_feedback_result.xml

```diff
@@ -84,44 +84,89 @@
             tools:visibility="visible">
 
             <View
-                android:id="@+id/line"
+                android:id="@+id/otherPredictionsSeparator"
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/feedback_result_line_height"
                 android:layout_marginLeft="@dimen/feedback_result_line_horizontal_margin"
                 android:layout_marginRight="@dimen/feedback_result_line_horizontal_margin"
                 android:background="@color/feedback_result_line"
             />
 
-            <TextView
+            <com.ciandt.dragonfly.example.components.classifications.ClassificationsView
+                android:id="@+id/otherPredictions"
```

Comment:
Same as above. 
What do you think?

![screenshot_20170823-102346](https://user-images.githubusercontent.com/68348/29568247-23ff6ce8-8726-11e7-9d97-0440d591ec78.png)